R/emissions_profile_any_compute_profile_ranking.R

```diff
@@ -42,7 +54,7 @@ check_emissions_profile_any_compute_profile_ranking <- function(data) {
 }
 
 rank_proportion <- function(x) {
-  rank(x) / length(x)
+  dense_rank(x) / length(unique(x))
 }
```

Comment:
I have applied `unique` here to remove extra counts due to duplicate values. For ex: 
Ranks: [3, 2, 2, 1]

Rank proportion without unique: [0.75, 0.5, 0.5, 0.25] (the ranks divided using 4)
Rank proportion with unique: [1, 0.67, 0.67, 0.33] (the ranks divided using 3)

Reply:
@maurolepore Thanks for suggesting the above alternatives! I think our `rank_proportion` function best suits our ranking use case.